contrib/udf/src/udaf_orthogonal_bitmap/bitmap_value.h

```diff
@@ -431,7 +430,7 @@ class Roaring64Map {
     bool isSubset(const Roaring64Map& r) const {
         for (const auto& map_entry : roarings) {
             auto roaring_iter = r.roarings.find(map_entry.first);
-            if (roaring_iter == roarings.cend())
+            if (roaring_iter == r.roarings.cend())
```

Comment:
Is it a serious bug?

Reply:
CRoaring has fixed it https://github.com/RoaringBitmap/CRoaring/blob/master/cpp/roaring64map.hh

Reply:
ok